src/fides/api/models/detection_discovery.py

```diff
@@ -277,6 +277,18 @@ class StagedResource(Base):
         server_default="{}",
         default=dict,
     )
+    user_assigned_data_uses = Column(
+        ARRAY(String),
+        nullable=False,
+        server_default="{}",
+        default=dict,
+    )
+    user_assigned_system_id = Column(
+        String,
+        ForeignKey(System.id_field_path),
+        nullable=True,
+        index=True,
```

Comment:
do we want an index here? it could make sense, though not sure it's strictly necessary. i'm good keeping it, but if so, we need to make sure we actually create/remove the index in the migration upgrade/downgrade

Reply:
Missed the migration in a merge apparently; added here: https://github.com/ethyca/fides/pull/5841/commits/b829ef4b83fe0c51bd913ba163941faadc1a08d1